automator/automator.py

```diff
@@ -363,9 +344,15 @@ def maybe_start_recording(self):
         subarrays = redis_util.suggest_recording(self.redis_server,
                                                  processing=self.processing)
         for subarray in subarrays:
-            log.info("subarray {} is ready for recording".format(subarray))     
-            redis_util.enable_recording(self.redis_server, subarray)       
-##            self.set_nshot(subarray, 1)
+            # If a primary sequence has ended, we don't want to record
+            # the next track.
+            if redis_util.primary_sequence_end(self.redis_server, subarray):
+                log.info(f"""A primary sequence has ended for {subarray},
```

Comment:
you only need the """ strings when you want it to be multi-line. for single line strings you can use single "

Reply:
I actually did this to break the line up so that the column width would be less than 80 characters; is this still a thing or does it not really matter?

Reply:
Oh, I see - that wasn't clear to me from the interface I was using. So if you want to break up a line to get below 80, the """ approach will add an extra \n to the string itself, which you don't want. You can either put a \ before the newline, or combine it like

```
("this sort of syntax"
 " will combine to one string in Python")
```

Reply:
OK done, thanks!